index.js

```diff
@@ -1,17 +1,60 @@
 class SortedList {
-  constructor() {}
+  constructor(items, length) {
+    this.items = [],
+    this.length =this.items.length
+  }
 
-  add(item) {}
+  add(item) {
+    const list = this.items.push(item)
+    list.sort(function(a, b) {
```

Comment:
Aqui seria poner en vez de list, this.items
```suggestion
    this.items.sort(function(a, b) {
```

Ademas que esta bien usando lo de function pero si te acostumbras a usar las Arrow Function (perdona por el uso del ingles suena pedante lo se, pero para que no te suene raro escucharlo) mejor